pyquil/api/_qpu.py

```diff
@@ -69,12 +69,14 @@ def _extract_bitstrings(ro_sources: List[Optional[Tuple[int, int]]],
 
 class QPU(QAM):
     @_record_call
-    def __init__(self, endpoint: str, user: str = "pyquil-user") -> None:
+    def __init__(self, endpoint: str, user: str = "pyquil-user", priority: int = 1) -> None:
```

Comment:
It may be worth noting somewhere in pyquil that 
`USER_PRIORITY =  1` should be the default for user-defined jobs, and
`BACKGROUND_PRIORITY = 2` should be the default for preemptible background jobs.

Otherwise the value of `1` seems a bit too arbitrary.